show/main.py

```diff
@@ -5,6 +5,7 @@
 import re
 
 import click
+from generic_config_updater.generic_updater import get_config_json_by_namespace
```

Comment:
>generic_updater [](http://example.com/codeflow?start=28&length=15)

I am wondering if this new dependency will recursively all the yang library as dependency. In theory, get running config does not require many.